docs/man/rpm-config.5.scd

```diff
@@ -209,6 +215,17 @@ for example:
 >>>
 ```
 
+## Hash algorithms
```

Comment:
Have an idea on *how* to reference these in the text (something we touched upon in our previous discussions): Emphasize the heading when mentioning the thing, for example:

```
The following _hash algorithms_ are known to rpm ...
```

We could do the same for the verification flags below.

If you agree, let's also update the man page template accordingly.

Reply:
Reason for the above suggestion: It's not obvious that the algorithms listed in the subsection on the bottom are what goes into this tunable.

Reply:
Oh, the intent here was to have `_HASHALGO_` in the "hash algorithms" section as a reference, just like `_VSFLAGS_` is there for the verification flags, I guess that got dropped at some editing round :unamused: 
A link to these values certainly needs to be there, I just prefer the shorter `_VSFLAGS_` style thing which is (presumably) easier to search for.

Reply:
Hmm. Aren't section names generally bolded instead of underlined?

Reply:
Added the `_HASHALGO_` reference to the "Hash algorithms" section to show the original intention.

Not opposed at all to adding emphasis on the section names (hash algorithms, verification flags) but maybe that should be done in a separate commit, updating the relevant places and also adding guidance on the type of emphasis on in the template. Right now the template just states "Use matching case and bold when referring to section names". Gut feeling is that bold is typically used when referring to section names, but there's no guidance that I can see in man-pages(7) and mixed usage (SYNOPSIS is several times non-bolded there etc). In any case, the spelling should match, it doesn't now...

Reply:
Oh indeed, `_HASHALGO_` is even better, I didn't notice we were using the same for the verification flags. So +1 on that.

As for bold vs. underline, I've looked at man-pages(7) too and indeed, there's no explicit mention of that. My thinking was that we (kinda) use underline when defining or mentioning *new terms* so it felt natural to also use that for referring to other sections. That said, bold is as good as any, so I don't have a strong preference here :smile: 

And yep, let's not dive into that here. Thanks for the fixup.